.gitignore

```diff
@@ -49,3 +49,5 @@ jspm_packages/
 # Target files
 lib/
 dist/
+
+apidocs
```

Comment:
ignore for committing makes sense. Should we make sure to include it in our files for npm though? How will these surface?

Reply:
Github pages?

Reply:
oh cool!